lib/toolbox/keys_dict.c

```diff
@@ -0,0 +1,324 @@
+#include "keys_dict.h"
+
+#include <storage/storage.h>
+#include <flipper_format/flipper_format.h>
+#include <toolbox/stream/file_stream.h>
+#include <toolbox/stream/buffered_file_stream.h>
+#include <toolbox/args.h>
+
+#define TAG "KeysDict"
+
+struct KeysDict {
+    Stream* stream;
+    size_t key_size;
+    size_t key_size_symbols;
+    size_t total_keys;
+};
+
+static inline void __add_ending_new_line(KeysDict* instance) {
```

Comment:
Please, avoid such naming. Even static functions should starts with lib name. This function should be names keys_dict_add_ending_new_line